CMakeLists.txt

```diff
@@ -23,14 +23,10 @@ add_library(roboteam_utils STATIC
         include/roboteam_utils/Position.h
         src/Mathematics.cpp
         include/roboteam_utils/Mathematics.h
-        #        src/Cone.cpp
-        #        include/roboteam_utils/Cone.h
```

Comment:
Cone is nice to keep. Tube was unused for the whole year, but as of yesterday it is used. I propose to keep this shape as well for the same reason.